pkgs/by-name/bs/bstone/package.nix

```
@@ -0,0 +1,48 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, gcc12
+, SDL2
+, libGL
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "bstone";
+  version = "1.2.12";
+
+  src = fetchFromGitHub {
+    owner = "bibendovsky";
+    repo = "bstone";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-wtW595cSoVTZaVykxOkJViNs3OmuIch9nA5s1SqwbJo=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    gcc12
+  ];
+
+  buildInputs = [
+    libGL
+    SDL2
+  ];
+
+  postInstall = ''
+    mkdir -p $out/bin
+    mv $out/bstone* $out/bin
+  '';
+
+  meta = with lib; {
```

```suggestion
  meta = {
```
nit: There's some [reasoning](https://github.com/NixOS/nixpkgs/pull/294383/files) for not going `with lib` on `meta`.